keps/sig-scheduling/2249-pod-affinity-namespace-selector/kep.yaml

```diff
@@ -0,0 +1,45 @@
+title: Namespace Selector for Pod Affinity
+kep-number: 2249
+authors:
+  - "@ahg-g"
+owning-sig: sig-scheduling
+participating-sigs:
+status: implementable
+creation-date: 2021-01-11
+reviewers:
+  - "@alculquicondor"
+  - "@Huang-Wei"
+  - "@liggitt"
+approvers:
+  - "@Huang-Wei"
+  - "@liggitt"
+prr-approvers:
+  - "@wojtek-t"
```

Comment:
Re-assign to me, Wojtek is over-subscribed

Reply:
done.